lib/datadog/tracing/contrib/propagation/sql_comment.rb

```diff
@@ -13,7 +13,7 @@ def self.annotate!(span_op, mode)
             return unless mode.enabled?
 
             # PENDING: Until `traceparent`` implementation in `full` mode
-            # span_op.set_tag(Ext::TAG_DBM_TRACE_INJECTED, true) if mode.full?
+            span_op.set_tag(Ext::TAG_DBM_TRACE_INJECTED, true) if mode.full?
           end
```

Comment:
I... must admit am not a fan of leaving this enabled.

I see that we're developing this feature in a branch, which may mean we'll eventually fix this before merging to master, but sometimes things don't go according to plan -- maybe someone on the team gets sick and the work needs to be picked up by someone else, and that other person will not have the context of "this is actually a placeholder and shouldn't be merged".

So overall while I think the benchmarking looks reasonable, my suggestion is to undo all the changes to production code, e.g. the `sql_comment.rb` file, until they are ready to go.

Reply:
@ivoanjo No worries, I will make sure the changes commented out before the merge.